### PR TITLE
MAPEX-203 Fix search engine 404 errors

### PR DESCRIPTION
**Pull request by allisonrobbins**
Add pages for "nonmappable" techniques so all techniques have a page on site. Will reduce 404 page not found errors when coming from google

### CODE REVIEW

File: src/mappings_explorer/site_builder.py [hunk 1]
```diff
@@ -1364,6 +1364,25 @@ def build_attack_pages(projects: list, url_prefix: str, breadcrumbs: list):
                 breadcrumbs=breadcrumbs,
                 non_mappables=non_mappables,
             )
+            # Build technique pages that don't have mappings to fix any linking errors
+            for technique in non_mappables:
```
Comment (mehaase) [inline src/mappings_explorer/site_builder.py:1368]: I was a little thrown off by the use of the term "non-mappable", which I thought applied to security capabilities and not ATT&CK techniques. Looking at the code above this I can see that this is doing the right thing, but for future improvements let's come up with a different term, e.g. "unmapped" versus "non-mappable".

**Review by mehaase — APPROVED** [review]
Approved but left a comment on something that was confusing to keep in mind for future work.

**Comment by mehaase** [comment]
@allisonrobbins please merge this when you're ready

File: src/mappings_explorer/site_builder.py (hunk 1 above)
Comment (allisonrobbins) [reply src/mappings_explorer/site_builder.py]: @mehaase that's a good point! I changed the name of those variables for clarity

**Pull request merged**